docs/docs/Templates/basic-prompting.md

```
@@ -23,34 +24,36 @@ This article demonstrates how to use Langflow's prompt tools to issue basic prom
+The model responds according to the prompt **Template** constructed in the **Prompt** component.
 
 4. To examine the **Template**, in the **Prompt** component, click the **Template** field.
 
 ```text
 Answer the user as if you were a GenAI expert, enthusiastic about helping them get started building something fresh.
 ```
 
-5. To create an environment variable for the **OpenAI** component, in the **OpenAI API Key** field, click the <Icon name="Globe" aria-label="Globe icon" /> **Globe** button, and then click **Add New Variable**.
+5. Add your **OpenAI API key** to the **Language model** component.
+
+	Optionally, create a [global variable](/configuration-global-variables) for the **OpenAI API key**.
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)